data/generic.yaml

```diff
@@ -597,11 +597,8 @@ annocheck:
     #
     # This section is optional.  If no annocheck tests are defined
     # here, rpminspect will skip the annocheck inspection.
-    #
-    # These job entries may be listed here or under a block named
-    # 'jobs:' if you prefer.
     jobs:
-        - hardened: --ignore-unknown
+        hardened: --ignore-unknown
```

Comment:
tl;dr: it's the difference between a dict and a list of dicts.

`foo: bar` is a dict (YAML calls it a "mapping").  The JSON equivalent would be `{ "foo": "bar" }`.

In JSON, we write arrays like `["foo", "bar"]`.  The YAML equivalent of this (called a "sequence", here in flow) would be:

```yaml
- foo
- bar
```

In other words, the leading dashes indicate that it's an array, and each line is an element of the array.  Without the dash, it's not an array but a dictionary:

```yaml
foo: bar
baz: buzz
```

is equivalent to the JSON

```json
{
    "foo": "bar",
    "baz": "buzz"
}
```

Having the dashes turns it into an array, producing an array of dictionaries instead of a single unified dictionary:

```yaml
- foo: bar
- baz: buzz
```

is the JSON

```json
[
    { "foo": "bar" },
    { "baz": "buzz" }
]
```

which has an extra level of indirection.

Reply:
Ah, ok, that makes sense.  I don't like YAML's vocabulary for this.